front/pages/api/stripe/webhook.ts

```diff
@@ -500,6 +480,23 @@ async function handler(
                 await sendReactivateSubscriptionEmail(adminEmail);
               }
             }
+          } else if (stripeSubscription.status === "active") {
```

Comment:
NIT: we could do `else if (stripeSubscription.status === "active" && subscription.trialing)`

Reply:
We don't fetch subscription so we don't have it